collector.go

```diff
@@ -126,8 +133,10 @@ func (c *Collector) Describe(ch chan<- *prometheus.Desc) {
 	ch <- c.subscriptionLastKnownEventNumber
 	ch <- c.subscriptionConnectionCount
 	ch <- c.subscriptionTotalInFlightMessages
+	ch <- c.subscriptionTotalNumberOfParkedMessages
 }
```

Comment:
oldest parked message metric should also be added here

Reply:
added: https://github.com/marcinbudny/eventstore_exporter/pull/14/commits/d0297a4e6c7912947714122f5a80c937856b067b